api/v1alpha5/conversion.go

```diff
@@ -237,6 +237,6 @@ func Convert_v1alpha5_OpenStackMachineSpec_To_v1alpha7_OpenStackMachineSpec(in *
 }
 
 func Convert_v1alpha7_APIServerLoadBalancer_To_v1alpha5_APIServerLoadBalancer(in *infrav1.APIServerLoadBalancer, out *APIServerLoadBalancer, s conversion.Scope) error {
-	// Provider have been added in v1alpha7 but have no equivalent in v1alpha5
+	// Provider was orignally added in v1alpha7, but was backported to v1alpha6, but has no equivalent in v1alpha5
```

Comment:
linter is complaining about this typo
```suggestion
	// Provider was originally added in v1alpha7, but was backported to v1alpha6, but has no equivalent in v1alpha5
```

Reply:
It let me add that commit! Not done that before. Lets see if it makes the tests pass. You'll want to squash it anyway.